Session/Conversations/Settings/ThreadSettingsViewModel.swift

```diff
@@ -1829,6 +1822,42 @@ class ThreadSettingsViewModel: SessionTableViewModel, NavigatableStateHolder, Ob
         }
     }
     
+    private func pinConversation(threadPinnedPriority: Int32) {
+        let isPinning: Bool = (threadPinnedPriority <= 0)
```

Comment:
The naming of these two confused me a little as I thought you call this function to pin the conversation (and then there would be a different one to unpin the conversation)

Renaming things might help improve the clarity?
• `togglePinnedStatus`
• `currentPinnedPriority`
• `isCurrentlyPinned`